geopmpy/geopmpy/hash.py

```diff
@@ -5,13 +5,10 @@
 
 
 from geopmdpy.gffi import gffi
-from geopmdpy.gffi import get_dl_geopm
+from geopmdpy.gffi import get_dl_geopmd
 
-gffi.cdef("""
-uint64_t geopm_crc32_str(const char *key);
-""")
 try:
-    _dl = get_dl_geopm()
+    _dl = get_dl_geopmd()
```

Comment:
The hash.py module is in geopmpy for historical reasons.  Maybe it is time to move it into geompdy? Shouldn't block this PR, but maybe we should open an issue.

Reply:
I opened https://github.com/geopm/geopm/issues/3620 for this